### PR TITLE
Access control: further reduce access control feature toggle checks

### PR DESCRIPTION
**Pull request by IevaVasiljeva**
**What this PR does / why we need it**:
reduces the amount of checks of whether `accesscontrol` feature toggle is set. This will make it easier to switch from `accesscontrol` feature toggle to a separate RBAC section in Grafana configuration.

**Which issue(s) this PR fixes**:

Related to https://github.com/grafana/grafana-enterprise/issues/2503

### CODE REVIEW

File: pkg/services/sqlstore/sqlstore.go [hunk 1]
```diff
@@ -451,6 +451,11 @@ func InitTestDBWithMigration(t ITestDB, migration registry.DatabaseMigrator, opt
 	return store
 }
 
+func InitTestDBWithAC(t ITestDB) *SQLStore {
+	store := InitTestDB(t, InitTestDBOpt{FeatureFlags: []string{featuremgmt.FlagAccesscontrol}})
+	return store
+}
+
```
Comment (kalleep) [inline pkg/services/sqlstore/sqlstore.go:458]: Do we really need this one? We can just pass feature flags in options

**Review by kalleep — COMMENTED** [review]
Looks good. Added some comments and i think we can remove features from both folder and dashboard service but we can do it in another pr

File: pkg/services/accesscontrol/accesscontrol.go [hunk 2]
```diff
@@ -222,3 +224,7 @@ func extractPrefixes(prefix string) (string, string, bool) {
 	attributePrefix := rootPrefix + parts[1] + ":"
 	return rootPrefix, attributePrefix, true
 }
+
+func IsDisabled(cfg *setting.Cfg) bool {
```
[the remark this answers is not in the document]
Comment (IevaVasiljeva) [reply pkg/services/accesscontrol/accesscontrol.go:228]: Yes, exactly. Mostly for SQL store methods, as injecting access control in SQL store proved to be a lot of work.

**Review by kalleep — APPROVED** [review]
LGTM

**Pull request merged**